src/stories/layouts/Wizard/Wizard.stories.js

```diff
@@ -13,36 +14,38 @@ export default {
 
 const Template = (args) => {
   return (
-    <Wizard {...args}>
-      <div className="step-1">
-        <h3>Step 1</h3>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens mnesarchum,
-                ei periculis adipiscing per, probo populo nec ad.
-      </div>
-      <div className="step-2">
-        <h3>Step 2</h3>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens mnesarchum,
-                ei periculis adipiscing per, probo populo nec ad.
-      </div>
-      <div className="step-3">
-        <h3>Step 3</h3>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens mnesarchum,
-                ei periculis adipiscing per, probo populo nec ad.
-      </div>
-      <div className="step-4">
-        <h3>Step 4</h3>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens mnesarchum,
-                ei periculis adipiscing per, probo populo nec ad.
-      </div>
-    </Wizard>
+    <MemoryRouter initialEntries={['/user/Pl3WJYa7vQ1ALVt0rHRV/study/108']}>
```

Comment:
I am not sure what these hardcoded values mean, but the rest of the code looks good to me. I pulled the branch and checked the steps, it seems to be working great.

Reply:
These are just values that mock the URL parameters (`experimentId` and `participantId`). We need this in the Storybook story to avoid crashing the app when we use `useParams()`